lexos/processors/prepare/scrubber.py

```diff
@@ -461,32 +461,25 @@ def get_remove_punctuation_map(
     return text, remove_punctuation_map
 
 
-def get_remove_digits_map() -> Dict[int, type(None)]:
-    """Get the digit removal map.
+def get_remove_digits(text: str) -> str:
+    """Removes all digits.
 
-    :return: A dictionary that contains all the digits that should be removed
-        mapped to None.
+    :param text: A unicode string representing the whole text that is being
+        manipulated.
+    :return: A string with all digits removed
     """
 
-    # Map of digits to be removed
-    try:
-        remove_digit_map = load_character_deletion_map(
-            constants.CACHE_FOLDER, constants.DIGIT_MAP_FILENAME)
-
-    except FileNotFoundError:
-        # If the digit map does not already exist, generate it with all
-        # unicode characters that start with the category 'N'
-        # See http://www.fileformat.info/info/unicode/category/index.htm for
-        # the list of categories
-        remove_digit_map = dict.fromkeys(
-            [i for i in range(sys.maxunicode)
-             if unicodedata.category(chr(i)).startswith('N')])
+    # Using "d" to represent a digit, character, "."
+    # to represent any unicode character used to indicate
+    # a decimal number, and "***" to represent any sequence of
+    # any digits, this pattern will match:
+    # 1) ***
+    # 2) ***.***
+    pattern = re.compile(r"(\d)|((?<=\d)[\u0027|\u002C|\u002E|\u00B7"
```

Comment:
we should capture as much as possible, so i think we want (\d**+**) [add + for runs of at least one (unicode) digit ... as well at the end of the regex \d**+**)